src/hooks/use-network-connection.ts

```diff
@@ -0,0 +1,6 @@
+import { useContext } from "react";
+import { NetworkConnectionContext } from "../providers/network-connection/network-connection-context";
+
+export default function useNetworkConnection() {
```

Comment:
For consistency, we've been using named exports across the board in our OSS repos. Mind updating this to `export function useNetworkConnection()...`, or exporting it after its definition?

```ts
function useNetworkConnection() { ... }

export { useNetworkConnection };
```

Additionally, I'm not sure if the intention is to add more logic in here, but this could be reduced with a function body:

```ts
const useNetworkConnection = () => useContext(NetworkConnectionContext);
```